fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -1651,6 +1606,7 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 	FlowLock fetchSemaphore;
 	int numFetchWaiters;
 	std::shared_ptr<ReadIteratorPool> readIterPool;
+	std::vector<Future<Void>> actors;
```

Comment:
The common pattern is to use an `actorCollection` for this purpose, e.g., code from `masterserver.actor.cpp`.

```
PromiseStream<Future<Void>> addActor;
Future<Void> collection = actorCollection(addActor.getFuture());
...
addActor.send(provideVersions(self));
addActor.send(serveLiveCommittedVersion(self));
...
try {
   loop choose {
      when(wait(collection)) { ... }
   }
} catch (Error& e) {
}
```

The advantage is that we can capture any errors in the collection. This can be done in another PR.